dts/arc/arc_axs.dtsi

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019, Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "skeleton.dtsi"
+
+#include <dt-bindings/i2c/i2c.h>
```

Comment:
i2c isn't supported yet, right? why include?

Reply:
I will remove it.